subprojects/docs/src/docs/userguide/publishingMaven.adoc

```diff
@@ -129,7 +131,7 @@ Maven restricts 'groupId' and 'artifactId' to a limited character set (`[A-Za-z0
 
 The only Unicode values that are explicitly prohibited are `\`, `/` and any ISO control character. Supplied values are validated early in publication.
 
-[[sec:customizing_the_generated_pom]]
+[[sec:modifying_the_generated_pom]]
```

Comment:
This reverts the link to what it was originally, correct? If so, :+1:

Reply:
It does! 🙂